Makefile.in

```diff
@@ -27,7 +28,11 @@ all:
 	done
 
 test: all
-	cd F-FrontEnd/src && bash ./compile_all_test.sh
+	@if [ $(WITH_NATIVE_FORTRAN_COMPILER) = "yes" ]; then\
+		cd F-FrontEnd/src && bash ./compile_all_test.sh;\
+	else\
+	    echo "Fortran tests disabled because of --without-native-fortran-compiler";\
+    fi
```

Comment:
The "fi" here is not very aligned is that fine?

Reply:
Syntactically it makes no difference, because the whole section is one string (see slashes at the end of each). Fixed it anyway, so it would look better visually.